### PR TITLE
Extract movie and movielist into own components.

### PR DESCRIPTION
**Pull request by cdaviduik**
First step towards decoupling the application.

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -10,69 +10,17 @@ const App = () => {
     const [error, setError] = useState(null);
     const [nominees, setNominees] = useState([]);
 
-
     function updateSearchQuery(e) {
         setSearchQuery(e.target.value);
     }
 
-    const DisplayBanner = () => {
```
Comment (cdaviduik) [inline src/components/App/App.js:18]: I didn't actually delete this, just move it to line 41.

File: src/components/App/App.js [hunk 2]
```diff
@@ -125,6 +84,14 @@ const App = () => {
             })
     }
 
+    const nominateMovie = movie => {
+        // nominate movie
+        console.log("nominate movie", movie);
```
Comment (KaraDaviduik) [inline src/components/App/App.js:89]: Should we remove the logging statement?

File: src/components/App/components/MovieList/components/Movie/Movie.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+const checkIfNominated = ({ movie, onNominate, nominees }) => {
+    if (!({ movie }.movie.Nominated)) {
+        nominateMovie({ movie, onNominate, nominees });
+    }
+}
+
+const nominateMovie = ({ movie, onNominate, nominees }) => {
+    if (nominees.length < 5) {
+        // setNominees(nominees => nominees.concat({ movie }));
```
Comment (KaraDaviduik) [inline src/components/App/components/MovieList/components/Movie/Movie.js:31]: Should we remove these comments?

**Review by KaraDaviduik — APPROVED** [review]

File: src/components/App/App.js (hunk 2 above)
Comment (cdaviduik) [reply src/components/App/App.js:89]: Probably should have, but will leave for now.

File: src/components/App/components/MovieList/components/Movie/Movie.js (hunk 3 above)
Comment (cdaviduik) [reply src/components/App/components/MovieList/components/Movie/Movie.js:31]: Eventually.

**Pull request merged**